pyproject.toml

```diff
@@ -29,13 +29,15 @@ controlnet-aux = {version = "^0.0.9", optional = true}
 lmdeploy = {version = "^0.5.3", optional = true}
 decord = {version = "^0.6.0", optional = true}
 sam-2 = {git = "https://github.com/landing-ai/segment-anything-2.git", branch = "main", optional = true}
+flash-attn = "1.0.5"
```

Comment:
Installing this dependency is tricky (I've seen in the past that poetry is not capable of handle it correctly). We should mark it as optional.

Reply:
Agree! Will add to README on the installation section what is needed to manually install flash-attn:
`poetry run pip install flash-attn`